idl/power.idol

```diff
@@ -98,5 +98,154 @@ Interface(
             ),
             idempotent: true,
         ),
+        "rendmp_dma_write": (
+            doc: "reads a DMA register from a Renesas multiphase power controller",
+            args: {
+                "addr": "u8",
+                "reg": "u16",
+                "data": "u32",
+            },
+            reply: Result(
+                ok: "()",
+                err: CLike("ResponseCode"),
+            ),
+            idempotent: true,
+        ),
+
+        // Low-level read/write APIs
+        //
+        // In all of these APIs, `index` is the index of the rail within
```

Comment:
We now have (I think?) at least three different ways of specifying a rail via Idol (and I personally have been very much part of the problem here); at some point, it would be really nice to consolidate these down to a single, canonical way of specifying this.

Reply:
Agreed, I opened https://github.com/oxidecomputer/hubris/issues/1342 to track.